src/patternfly/components/AboutModalBox/styles.scss

```diff
@@ -0,0 +1,267 @@
+  --pf-c-about-modal-box__close--PaddingBottom: var(--pf-c-about-modal-box__brand--PaddingBottom);
+  --pf-c-about-modal-box__close--PaddingRight: var(--pf-c-about-modal-box--PaddingRight);
+  --pf-c-about-modal-box__close--ZIndex: var(--pf-global--ZIndex--2xl);
+
+  // Brand img
+  --pf-c-about-modal-box__brand-image--Height: pf-size-prem(40px);
+
+  // Header
+  --pf-c-about-modal-box__header--PaddingRight: var(--pf-c-about-modal-box--PaddingRight);
+  --pf-c-about-modal-box__header--PaddingBottom: var(--pf-global--spacer--sm);
+  --pf-c-about-modal-box__header--PaddingLeft: var(--pf-c-about-modal-box--PaddingLeft);
+
+  // Strapline
+  --pf-c-about-modal-box__header-strapline--PaddingBottom: var(--pf-global--spacer--md);
+
+  // Content
+  // Using global padding values for margins to prevent scrollbar extending past content
+  --pf-c-about-modal-box__content--MarginTop: var(--pf-c-about-modal-box--PaddingTop);
+  --pf-c-about-modal-box__content--MarginRight: var(--pf-c-about-modal-box--PaddingRight);
+  --pf-c-about-modal-box__content--MarginBottom: var(--pf-c-about-modal-box--PaddingBottom);
+  --pf-c-about-modal-box__content--MarginLeft: var(--pf-c-about-modal-box--PaddingLeft);
+  // This is the min allowable height of the content
+  --pf-c-about-modal-box__content--MinHeight: calc(1rem * 4 * var(--pf-global--LineHeight--md));
+
+  // Logo
+  --pf-c-about-modal-box__logo--PaddingRight: var(--pf-c-about-modal-box--PaddingRight);
+  --pf-c-about-modal-box__logo--PaddingBottom: var(--pf-c-about-modal-box--PaddingBottom);
+  --pf-c-about-modal-box__logo--PaddingLeft: var(--pf-c-about-modal-box--PaddingLeft);
+
+  // Logo image
+  --pf-c-about-modal-box__logo-image--PaddingTop: var(--pf-global--spacer--md);
+  --pf-c-about-modal-box__logo-image--PaddingRight: var(--pf-global--spacer--md);
+  --pf-c-about-modal-box__logo-image--PaddingBottom: var(--pf-global--spacer--md);
+  --pf-c-about-modal-box__logo-image--PaddingLeft: var(--pf-global--spacer--md);
+  // height and width determined manually by visual design
+  --pf-c-about-modal-box__logo-image--Width: pf-size-prem(236px);
+  --pf-c-about-modal-box__logo-image--Height: pf-size-prem(110px);
+  --pf-c-about-modal-box__logo-image--BackgroundColor: $pf-color-white;
+
+  // Button close
+  --pf-c-about-modal-box__button-close--Color: var(--pf-global--Color--100);
+  --pf-c-about-modal-box__button-close--hover--Color: var(--pf-global--Color--100);
+  --pf-c-about-modal-box__button-close--ZIndex: var(--pf-global--ZIndex--2xl);
+  --pf-c-about-modal-box__button-close--FontSize: var(--pf-global--FontSize--xl);
+  // Height and width are 2x font-size
+  --pf-c-about-modal-box__button-close--Width: calc(var(--pf-c-about-modal-box__button-close--FontSize) * 2);
+  --pf-c-about-modal-box__button-close--Height: calc(var(--pf-c-about-modal-box__button-close--FontSize) * 2);
+  --pf-c-about-modal-box__button-close--BackgroundColor: $pf-color-black-1000;
```

Comment:
Please make a comment on why we are hardcoing this color and not consuming a global variable